pkg/kubectl/cmd/create_deployment.go

```diff
@@ -153,7 +165,18 @@ func createDeployment(f cmdutil.Factory, cmdOut, cmdErr io.Writer,
 	generatorName = fallbackGeneratorNameIfNecessary(generatorName, resourcesList, cmdErr)
 
 	imageNames := cmdutil.GetFlagStringSlice(cmd, "image")
-	generator, ok := generatorFromName(generatorName, imageNames, deploymentName)
+	replicas := cmdutil.GetFlagInt(cmd, "replicas")
+
+	command := []string{}
+	argsArray := args[1:]
```

Comment:
Can you verify what is in args? I suspect Cobra already take out the first arg for us, since I saw https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/apply.go#L143-L149